packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -559,8 +559,16 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
         selectAll?: boolean
     ) {
         const range = editor.getDocument().createRange();
-        if (selectAll) {
-            range.selectNodeContents(cell);
+        if (selectAll && cell.firstChild) {
+            const cellStart = cell.firstChild;
+            const cellEnd = cell.lastChild || cell.firstChild;
+            // Get first deepest editable position in the cell
+            const posStart = normalizePos(cellStart, 0);
```

Comment:
need null check, in case cellStart/cellEnd is null.

If that happens, maybe we need to create a paragraph? Or just simple put focus into the cell.

Reply:
They can not be null.

Reply:
We cannot make such assumption. At least we need a null check here or it can throw at runtime.

Reply:
The null check is above, after passing it cellStart/cellEnd can not longer be null.

Reply:
Oh I see, sorry I missed it.